src/main/resources/db.properties

```diff
@@ -0,0 +1,2 @@
+spring.datasource.username=postgres
+spring.datasource.password=postgres
```

Comment:
To deploy we need to be able to set the db user and password as environment variables. Is this also possible?
Same thin for the configuration of the database-URL.

Reply:
with the current solution its possible to override the default settings in the application.properties file. You have to add to the java -jar crossword-service-0.0.1-SNAPSHOT.jar command --spring.datasource.username=InsertUsernameHere and --spring.datasource.password=InsertPasswordHere. The same applies to all the setting set in the application.properties file.